tests/test_freqtradebot.py

```diff
@@ -2888,11 +2888,15 @@ def test_adjust_entry_maintain_replace(
     assert freqtrade.strategy.adjust_entry_price.call_count == 1
 
     # Check that order is replaced
+    # TODO replace order at a price that can't be fullfilled
```

Comment:
@xmatthias  How can I create an order that is not immediately filled? Probably by setting an entry price far from market price, But I'm not yet familiar with the mocking process of current market price, Any advise or example test? (looking for one at the moment)

Reply:
The problem here is a different one - one i'm not sure will be straightforward to fix as this PR (as it'll taint the actual changes).

`open_trade`  is incomplete - the INTEND is to have an open exit order - not an open order.
`open_trade_usdt` does this correctly - and will therefore be better suited for this test.